src/applications/personalization/dashboard/components/Dashboard.jsx

```diff
@@ -247,29 +260,73 @@ const Dashboard = ({
               )}
 
               {/* LOA3 user experience */}
-              {props.showClaimsAndAppeals && (
-                <DowntimeNotification
-                  dependencies={[
-                    externalServices.mhv,
-                    externalServices.appeals,
-                  ]}
-                  render={RenderClaimsWidgetDowntimeNotification}
-                >
-                  <ClaimsAndAppeals useLighthouseClaims={useLighthouseClaims} />
-                </DowntimeNotification>
-              )}
-              {isLOA3 && (
-                <>
-                  <HealthCare isVAPatient={isVAPatient} />
-                  <Debts />
-                  <BenefitPayments
-                    payments={payments}
-                    showNotifications={showNotifications}
-                  />
-                  <EducationAndTraining />
-                  <BenefitApplicationDrafts />
-                </>
-              )}
+              {/* Remove everything in <Toggler.Enabled> after maintenance is over */}
+              <Toggler
+                toggleName={Toggler.TOGGLE_NAMES.authExpVbaDowntimeMessage}
+              >
+                <Toggler.Enabled>
+                  <div className="vads-u-margin-top--4 vads-l-col--8">
+                    <VaAlert
+                      closeBtnAriaLabel="Close notification"
+                      closeable
+                      onCloseEvent={handleDismiss}
+                      status="warning"
+                      visible={dismissed !== 'true'}
+                      data-testid="downtime-alert"
+                    >
+                      <h2 slot="headline">
+                        We’re updating our systems right now
+                      </h2>
+                      <div>
+                        <p className="vads-u-margin-y--0">
+                          We’re updating out systems to add the 2024
+                          cost-of-living increase for VA benefits. If you have
+                          trouble using this tool, check back after{' '}
+                          <strong>Sunday, November 19, 2023</strong>, at{' '}
+                          <strong>7:00 p.m. ET</strong>.
+                        </p>
+                      </div>
+                    </VaAlert>
+                  </div>
+
+                  {isLOA3 && (
+                    <>
+                      <HealthCare isVAPatient={isVAPatient} />
+                      <EducationAndTraining />
+                      <SavedApplications />
```

Comment:
Did saved applications get removed from the imports?

Reply:
yes :( just pushed up a fix